Working Open statement (#3)
hyphacoop/handbook: working-open.md

# Working Open

> Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.

Reviewer: > (and exactly what)

Not sure how to read this. Do you mean the tools/spaces/docs we keep open? If so, might "(and where)" capture the sentiment more clearly?

Reviewer 2: ```suggestion
Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
```

## Why?

> - In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.

Reviewer 2: What does the phrase "new economy" refer to? It was historically used in relation to [internet related innovation](https://www.nber.org/papers/w7833), however, I think you might be referring to [something like this](https://neweconomy.net/about/what-is-the-new-economy)?

Reviewer: Good point. I'd be satisfied with a link out to something, rather than diving into the meaning in this doc itself, if that works for others

Reviewer: Or we could start a glossary in the handbook, and start corralling this stuff :)

Author: Yeah I was borrowing "new economy" as shorthand for the three aspects they mention: " just, sustainable, and democratic future" + solidarity economy etc... I think we can use a non-jargon word here (and maybe crib from our mission/vision), but think a glossary of things would be good? I'm filing an issue of glossary terms here --> #4

Author: Okay added suggested language below, I'm resolving this for now

> - In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.

Author: Proposed revision with language from our mission:
```suggestion
- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
```

> - Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.

Reviewer: <3 

Correction: *negotiating

> - Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.

Reviewer 2: ```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
```

> - Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy

Reviewer: *out our

> - Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy

Author: ```suggestion
- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy
```

# How and what?

- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken

Legend:

> 👀 - View only

Reviewer: Might it be more clear to annotate the "edit" access? I feel like the default assumption is that open is "viewable", so it's the "edit" escalation that's more clear to see noted :)

Reviewer: Or we could just be explicit, and have a notation for both?

👀 view-only
📝 view & edit

Reviewer 2: > at any point
Does this mean "for any reason" as well?

Author: I like the explicit double notation @patcon! will add in

Author: > > at any point
> 
> Does this mean "for any reason" as well?

@uditvira -- I'm not sure I'm seeing the crux of the difference here. Could you say more? 

I thought this captured the conversation we had: currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured.

Just to restate my vibes from the convo: in time we should revisit and set norms of what stays open + a more explicit consent process (to steal @patcon's wording) for the things that we've noticed as the sites where we need differential shading. I think over time we'll get to a point where we recognize when that happens. As we learn I think we'll need to have some discussion and reflection about the reasons we are okay with around privacy.

Reviewer 2: > at any point

... in time?

I assumed you were referring to an occasion (before, during, or after), which does not describe the scope of and limits to redactions

> currently working with this policy and there is no mechanism explicated for approving/rejecting a request so any request is honoured

This is the clarification I was looking for, thanks. Yes, I agree there is no need to systematize more explicitly for now

Author: Okay. If additional words would make things more clear @uditvira please propose a friendly amendment!

> 👀 - View only

Author: ```suggestion
👀  view only
📝  view and edit
```

**Default open to all**:

- Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
- Aggregated financal statements 👀

> - Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀

Reviewer: Is this repetition of "aggregated financial statements" intended to capture the separate org-wide vs project-level scope? If so, perhaps we could make this more clear

Author: It is intended to speak to project finances, can you suggest a wording to improve this?

Reviewer: - Scope: organization. Aggregate financial statements 👀
- Scope: project. Deliverables*, task tracking, meeting notes, and aggregated financial statements 👀

* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*

**Default open to members**:

- Meeting notes and documents (write-access)
- Task tracking, project management (PM), and decision making (DM) tools (write-access)
- Finances (budget, expenses, and cash flow projections) and audit records 👀
- Potential opportunities and leads
- Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀

**Default closed to working groups and project members**:

> - Full personal/financial information of members

Reviewer: Does this reflect a discussion about sharing salaries internally? As in, do we want to explicitly refer to salaries in this doc? (I missed one of these all-hands meetings)

Author: I don't think we have had a discussion about "sharing" salaries but we have had a discussion about setting salaries floors as well as a max pay differential ratio (e.g., folks paid minimum XX/hour; max difference 1:2), I don't think it would be unreasonable to be very explicit about salaries and our pay guidelines, but that is different than actually sharing someone's YTD earnings.

My intent with this line (and I think a reflection of discussion about personal and financial information) what that it was for things like: home addresses, bank accts, SIN, health stuff (say if we have benefits and there is some minimum coordination of knowing OHIP details, or say a leave, or terrible case someone gets injured on the job... not like actually anyones health data), or also potential "HR" info. 

Some (scant) links on this: 
https://www.priv.gc.ca/en/privacy-topics/privacy-at-work/
https://www.minkenemploymentlawyers.com/employment-law-issues/workplace-privacy-an-increasingly-important-issue-in-the-information-age/

Reviewer: Ah cool! Thanks! Adding that "for example" bit would def clarify this

Author: added!

> - Full personal/financial information of members

Author: ```suggestion
- Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)
```

- Personal/financial information involving external parties
- Administration privileges including financal for services and infrastructure
- Project sensitive data sets (especially all non-anonymized/identifiable) and critical infrastructure

**Require consent before releasing**:

> - Audio, visual (photography) and video recordings

Reviewer: Curious if the essence of this is visual representation of someone... As in, is it mostly likely that someone would allow a recording, so long as their face isn't present? Happy to defer this, but would like to eventually discuss framing this to model how folks usually conceptualize progressive consent to transparency :)

- audio recording of spoken word
- image or video representation of person

Author: Can you link to a resource on this? Super curious if there is intentionally some implicit degree difference by placing these as separate?

I was not thinking the essence was visual representation, with tri-council research ethics and in my previous work, the emphasis has been on ability to re-identify. Audio can be more identifiable than an image (in my experience). In terms of progressive consent-- the bullets you suggest doesn't reflect my experience // research consent forms and convos I've been in.

Happy to chat more but I think this might be a little premature?

Reviewer: Hm. So your experience is not that images are perceived as more sensitive than audio? I've witnessed people avoid being on camera, but will ask questions from the audience knowing their voice is livestreamed. I'm not sure whether it meaningfully makes a difference in ethics, as both are perhaps equally likely to re-identify, but I've understood visual image to be more closely guarded than audio.

Where the rubber hits the road on this is that I can imagine someone consenting to audio of meeting being shared, but not video. I'm interested in the perceived baby-steps that might allow people to feel secure and in-control with sharing. Also, re-identificiation can happen from a verbatim transcript via [stylometric analysis](https://en.wikipedia.org/wiki/Stylometry), but I definitely have an assumption that most people feel a great deal more comfortable sharing a written transcript.

Sorry, no resources to share here, just my own perceptions.  Though Audrey Tang does have a [transparency policy](https://talk.pdis.nat.gov.tw/t/principles-for-handling-official-visits-to-digital-minister-audrey-tang/70) that favours at least one artifact be made public by default -- written, audio, or video -- but allows that to be chosen by person, presumably based on level of comfort and preference.

Author: I'm speaking from a (narrow) research context, but yeah wrt to photos v. audio. Video is a much higher bar and folks perception of that has many different facets. 

I'm going to resolve this here because I'v linked from #5 -- we'll definitely need to pick this up later, maybe with a project on deck? to help navigate

> - Audio, visual (photography) and video recordings

Author: ```suggestion
- Audio, visual, and video recordings (e.g., photographs or recorded interviews)
```

# Working Open

Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.

## Why?

> - In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.

Reviewer 3: ```suggestion
- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods
```

> - In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.

Reviewer: This line is getting at something like "other people can read and learn from our stuff", right? Having feels about [thing-explainer'ing](https://xkcd.com/thing-explainer/) some of this language, but don't want to be pedantic! :)

Wondering if conceiving an "audience" or some personas for this handbook would be helpful. Our parents? Children? A baby boomer investor? Tradespersons?

Author: This is our mission/vision statement.

Author: I'm going to leave this for now because we are our initial audience and this is our own mission statement! I've added your thoughts to #8

> - Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.

Reviewer 3: ```suggestion
- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions
```

Author: You'd like to remove the periods?

Author: Reviewed and we err on the side of periods at the end of full sentences, including bullet points, both in this doc and in the other committed material. I'd rather add the one missing period than remove all of these

Reviewer: I ever-so-slightly prefer punctuation on bullets too :) Esp with markdown. If you agree, perhaps mark these as resolved @benhylau?

Reviewer 3: @dcwalk I have no preference other than being consistent.

Author: Okay, resolving these suggestions then

- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy

# How and what?

> - We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken

Reviewer 3: ```suggestion
- We currently work with an [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
```

> - We currently work with an [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken

Reviewer: Re-reading. Perhaps we could strike the use of "in camera"? It's a historic but perpetually-confusing term that means the opposite of what contemporary people assume. Here's an attempt to trim/simplify.

```suggestion
- For meeting notes, we accommodate redactions and going off-the-record. Participants can request to stop taking notes, or redact and correct notes after they have been taken.
```

Reviewer: Members => participants, because this isn't just for members, right?

Author: I'm not sure about other folks, but if anyone has attended a more formally chaired meeting than an in camera process is not likely unfamiliar. I'm fine with clarifying language, I guess I wonder if we didn't already address your concerns with the following statements?

I'd also prefer we keep the temporal aspects: that this is our current practice.

Legend:  
👀  view only  
📝  view and participate  

> **Default open to all**:

Reviewer: This might be more explicit:

```suggestion
**Default open to public**:
```

- Scope: Organization
  - Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀

>   - Aggregated financal statements 👀

Reviewer 3: ```suggestion
  - Aggregated financial statements 👀
```

- Scope: Project
  - Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀

> * *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*

Reviewer 3: ```suggestion
* *This would include source repositories and code (e.g. on GitHub) and other process materials based on consensus on licensing and usage constraints.*
```

**Default open to members**:

- Scope: Organization
  - Meeting notes and documents 📝 
  - Task tracking, project management (PM), and decision making (DM) tools 📝 
  - Finances (budget, expenses, and cash flow projections) and audit records 👀
  - Potential opportunities and leads 📝 
- Scope: Project

>   - Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀

Reviewer 3: ```suggestion
  - Project proposal(s), statement(s) of work (in-progress/signed), in-progress deliverables, and finances (budget, expenses, and cash flow projections) 👀
```

**Default closed to specific working groups and project members**:

- Scope: Organization

>   - Full personal/financial information of members (e.g., home addresses, banking information, and social insurance numbers)

Reviewer 3: ```suggestion
  - Full personal/financial information of members (e.g. home addresses, banking information, and social insurance numbers)
```

Author: I favour commas after e.g./i.e., it isn't a typo but a rather a distinct usage, see discussion: https://www.dailywritingtips.com/comma-after-i-e-and-e-g/ (I'd also say it fits in this text)

Reviewer 3: Not saying this one is a typo, but I think without the comma is more common and so more likely to be consistently used. Same with 1 or 2 spaces after periods, etc. If you prefer having the comma we can aim for that in all our documents, but I definitely feel not having it is likely to stay consistent.

Reviewer: We could use [`vale`](https://errata-ai.github.io/vale/) and [dangerCI gem](https://danger.systems/guides/what_does_danger_do.html) to encode our rules and do some of these evolving checks. Demo: https://github.com/patcon/dangerfile-test/pull/1

Style rule: https://github.com/patcon/dangerfile-test/blob/master/ci/vale/styles/HyphaCoop/Eg.yml

Reviewer: I'm pretty indifferent about which, though I'll need to get used to using commas. But I'm fine with whatever conventions as long as we're on same page :)

(hence my appreciating the bot auto-correcting)

Author: Commas after "For example" (e.g.) and "Further" (i.e.) tends to be grammatically correct when used without abbreviations. In my experience, but this is based on a lot of academic reading I recognize, commas are the more frequent usage.

  - Personal/financial information involving external parties 📝 

>   - Administration privileges including financal for services and infrastructure 📝 

Reviewer 3: ```suggestion
  - Administration privileges including financial for services and infrastructure 📝 
```

- Scope: Project
  - Project sensitive data sets (especially all non-anonymized/identifiable) and critical infrastructure 📝 

**Require consent before releasing**:

> - Audio, visual, and video recordings (e.g., photographs or recorded interviews)

Reviewer 3: ```suggestion
- Audio, visual, and video recordings (e.g. photographs or recorded interviews)
```
